src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -155,3 +156,13 @@ export const sumMainnetTokensUsdBalancesPerNetwork = ({
  */
 export const filterEnabledTokens = ([$tokens]: [$tokens: Token[]]): Token[] =>
 	$tokens.filter((token) => ('enabled' in token ? token.enabled : true));
+
+/** Pins enabled tokens at the top of the list, preserving the order of the parts.
+ *
+ * @param $tokens - The list of tokens.
+ * @returns The list of tokens with enabled tokens at the top.
+ * */
+export const pinEnabledTokensAtTop = <T extends Token>(
+	$tokens: TokenToggleable<T>[]
+): TokenToggleable<T>[] =>
+	$tokens.sort((a, b) => (a.enabled === b.enabled ? 0 : a.enabled ? -1 : 1));
```

Comment:
Just an idea: `.sort(({enabled: a, {enabled: b}}) => Number(b) - Number(a))`

Note: note sure about b - a or a - b ;)

Reply:
that is correct! changed